includes/LoadWizards/WindLoadWizard.js

```diff
@@ -0,0 +1,242 @@
+    this.windLoadWizard.base_corner_nodes = base_corner_nodes;
+    this.windLoadWizard.roof_corner_nodes = roof_corner_nodes;
+    for (var i = 0; i < load_cases.length; ++i) {
+        this.windLoadWizard.generate_into_load_cases[i + 1].load_case = load_cases[i];
+    }
+};
+
+/**
+ * Creates flat/monopitch roof
+ * @param {Number}  no                  Wind load wizard index, can be undefined
+ * @param {Array}   roof_corner_nodes   Roof corner nodes indexes
+ * @param {Array}   load_cases          Load cases
+ * @param {String}  comment             Comment, can be undefined
+ * @param {Object}  params              Additional parameters, can be undefined
+ */
+WindLoadWizard.prototype.RoofMonopitch = function (no,
+    roof_corner_nodes,
+    load_cases,
+    comment,
+    params) {
+    ASSERT(typeof roof_corner_nodes !== "undefined" && roof_corner_nodes.length === 4, "Four roof corner nodes must be specified");
+    ASSERT(typeof load_cases !== "undefined" && load_cases.length === 8, "Eight load cases must be specified");
+    this.windLoadWizard = createWindLoadWizard(no, comment, params);
+    this.windLoadWizard.type = wind_loads.TYPE_ROOF_MONOPITCH;
+    this.windLoadWizard.roof_corner_nodes = roof_corner_nodes;
+    for (var i = 0; i < load_cases.length; ++i) {
+        this.windLoadWizard.generate_into_load_cases[i + 1].load_case = load_cases[i];
+    }
+};
+
+/**
+ * Creates duopitch roof
+ * @param {Number}  no                  Wind load wizard index, can be undefined
+ * @param {Array}   roof_corner_nodes   Roof corner nodes indexes
+ * @param {Array}   load_cases          Load cases
+ * @param {String}  comment             Comment, can be undefined
+ * @param {Object}  params              Additional parameters, can be undefined
+ */
+WindLoadWizard.prototype.RoofDuopitch = function (no,
+    roof_corner_nodes,
+    load_cases,
+    comment,
+    params) {
+    ASSERT(typeof roof_corner_nodes !== "undefined" && roof_corner_nodes.length === 6, "Six roof corner nodes must be specified");
+    ASSERT(typeof load_cases !== "undefined" && load_cases.length === 12, "Twelve load cases must be specified");
+    this.windLoadWizard = createWindLoadWizard(no, comment, params);
+    this.windLoadWizard.type = wind_loads.TYPE_ROOF_DUOPITCH;
+    this.windLoadWizard.roof_corner_nodes = roof_corner_nodes;
+    for (var i = 0; i < load_cases.length; ++i) {
+        this.windLoadWizard.generate_into_load_cases[i + 1].load_case = load_cases[i];
+    }
+};
+
+/**
+ * Sets wind perpendicular to roofs
+ * @param {Array}   roof_sides_accessibility   Roof sides accessibility
+ *                                             [Wall 1, Wall 2, Wall 3, Wall 4] (Vertical walls with flat/monopitch roof, Vertical walls with duopitch roof)
+ *                                             [Direction 1, Direction 2, Direction 3, Direction 4] (Flat monopitch roof, Duopitch roof)
+ */
+WindLoadWizard.prototype.Direction = function (roof_sides_accessibility) {
```

Comment:
This would be good to have in the main function - together with definition of load cases - it has an influence